tests/unit/sagemaker/image_uris/test_autogluon.py

```diff
@@ -57,6 +57,8 @@
     "0.6.1",
     "0.6",
     "0.6.2",
+    "0.7.0",
+    "0.7",
```

Comment:
Not sure if the tests will pass: this line needs to be updated since python version changed
```python
    py_version = "py37" if version == "0.3.1" else "py38"
```